src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -51,6 +53,8 @@
 @AutoValue
 public abstract class SampleTransformer {
 
+  private static final int DOC_LINE_MAX_WIDTH = 80;
```

Comment:
Should this be overridable by language? IIUC, the Java convention is 100 chars. Go isn't strict about line length, so we could also probably do 100 chars. Don't know about other languages.

Reply:
I didn't find any requirements by other languages. Let's just do 100 here for now? And we can always change it if some language has a different requirement in the future.
@beccasaurus WDYT?

Reply:
FWIW, Google style guides (internal and external)  for other languages, last time I checked, seem to go for an 80 char limit (eg https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length)

Reply:
OMG this is so annoying :( Yeah I'll make this value override-able

Reply:
Humans are so picky. "What do you mean, 81 columns? Can't read!" Sigh....

Thanks for doing this! Feel free to do it in a chaser PR if it's simpler.